elisp/intero.el

```diff
@@ -624,7 +624,19 @@ May return a qualified name."
   "Return buffer-file-name stripped of any text properties."
   (let ((name (buffer-file-name buffer)))
     (when name
-      (substring-no-properties name))))
+      (canonicalize-path (substring-no-properties name)))))
+
+(defun canonicalize-path (path)
```

Comment:
It's not usual to expose top-level functions without a module prefix (like `-intero`), but don't worry I'll add that before merging!